### DISCUSSION TITLE
how to uninstall

### DISCUSSION THREAD

**Discussion by techhound21** [SelfControlApp/selfcontrol#704] [Q&A]
Can someone please provide steps to completely remove SelfControl from my system?

**Comment by cstigler** [comment]
We do get this question a bunch, so I just added an entry to the FAQ: https://github.com/SelfControlApp/selfcontrol/wiki/FAQ#q-how-do-i-uninstall-selfcontrol

**Reply by jessegilbride** [reply]
If a user does the no-no thing and uninstalls SelfControl while a block is running, can SelfControl be reinstalled to remove the block(s)?

FYI, I haven't used the software, although considering it, and I'm concerned about the edge case of the premature uninstall doomsday scenario.

**Reply by cstigler** [reply]
If you just deleted the app from your Applications folder, yep you can redownload it and it'll keep working. If you tried to run command-line scripts you found on the internet to remove it, all bets are off - it'll probably be fine, but depends on what you did.
